data_model/src/events/notification.rs

```diff
@@ -113,8 +113,8 @@ pub mod model {
     #[ffi_type]
     #[non_exhaustive]
     pub enum NotificationEventFilter {
```

Comment:
why is this filter here and not in `filter.rs`?

Reply:
AFAIU, `filters.rs` is located in the `data` module, so it's only for data events.

```
events
├── data
│   ├── events.rs
│   ├── filters.rs <-  data event filters go here
│   └── mod.rs
├── execute_trigger.rs
├── mod.rs
├── notification.rs <- both notification events & notification event filters go here
├── pipeline.rs <- same for pipeline events & filters
└── time.rs
```